PE-23 adding PR template and readme (#1)
Lodgelink-Inc/.github: README.md

# .github

> ## This repository contains organization-wide GitHub configurations for shared use across Lodgelink-Inc repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.

Reviewer: ```suggestion
## This repository contains organization-wide GitHub configurations for shared use across the organizations repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.
```

> ## This repository contains organization-wide GitHub configurations for shared use across the organizations repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.

Reviewer 2: ```suggestion
## This repository contains organization-wide GitHub configurations for shared use across the organization's repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.
```

> ## This repository contains organization-wide GitHub configurations for shared use across the organizations repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.

Reviewer: ```suggestion
## This repository contains organization-wide GitHub configurations for shared use across repositories. It helps maintain consistency, reduce duplication, and streamline development workflows.
```
